common/lib/common-utils/src/typedEventEmitter.ts

```diff
@@ -66,4 +123,6 @@ export class TypedEventEmitter<TEvent>
     readonly prependOnceListener: TypedEventTransform<this, TEvent>;
     readonly removeListener: TypedEventTransform<this, TEvent>;
     readonly off: TypedEventTransform<this, TEvent>;
+
+    readonly emit: EventEmitSignatures<this, TEvent>;
```

Comment:
Question for folks - Should I do this PR against next? Seems like it. 

But I should be able to bump client's dep on common-utils in main? Well, maybe not because I believe Aqueduct DataObject extends TypedEventEmitter, so this breaking change would be exposed there... sound right?